### PR TITLE
Zminteg

### PR DESCRIPTION
**Pull request by zohomail**
---
### When submitting a connector, please make sure that you follow the requirements below, otherwise your PR might be rejected. We want to make you have a well-built connector, a smooth certification experience, and your users are happy :) 

If this is your first time submitting to GitHub and you need some help, please sign up for this [session](https://forms.office.com/pages/responsepage.aspx?id=KtIy2vgLW0SOgZbwvQuRaXDXyCl9DkBHq4A2OG7uLpdUMTFJWFFGVUxBNUFZQjZWRUdaOE5BMFkwNS4u). 

- [x] I attest that the connector doesn't exist on the Power Platform today. I've verified by checking the pull requests in GitHub and by searching for the connector on the platform or in the documentation. 
- [x] I attest that the connector works and I verified by deploying and testing all the operations. 
- [x] I attest that I have added detailed descriptions for all operations and parameters in the swagger file.
- [x] I attest that I have added response schemas to my actions, unless the response schema is dynamic. 
- [x] I validated the swagger file, `apiDefinition.swagger.json`, by running `paconn validate` command.
- [x] If this is a certified connector, I confirm that `apiProperties.json` has a valid brand color and doesn't use an invalid brand color, `#007ee5` or `#ffffff`. If this is an independent publisher connector, I confirm that I am not submitting a connector icon.

If you are an Independent Publisher, you must also attest to the following to ensure a smooth publishing process:
- [ ] I have named this PR after the pattern of "Connector Name (Independent Publisher)" ex: HubSpot Marketing (Independent Publisher)
- [ ] Within this PR markdown file, I have pasted screenshots that show: 3 unique operations (actions/triggers) working within a Flow. This can be in one flow or part of multiple flows. For each one of those flows, I have pasted in screenshots of the Flow succeeding. 
- [ ] Within this PR markdown file, I have pasted in a screenshot from the Test operations section within the Custom Connector UI.
- [ ] If the connector uses OAuth, I have provided detailed steps on how to create an app in the readme.md.

### CODE REVIEW

File: certified-connectors/Zoho ZeptoMail/readme.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# Zoho ZeptoMail
```
Comment (Amjed-Ayoub) [inline certified-connectors/Zoho ZeptoMail/readme.md:1]: @zohomail can you please include all the required section from our template [here ](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md)and for code raw view [here](https://raw.githubusercontent.com/microsoft/PowerPlatformConnectors/dev/templates/certified-connectors/readme.md).

File: certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json [hunk 2]
```diff
@@ -0,0 +1,800 @@
...
+                "message": {
+                  "type": "string",
+                  "description": "message"
+                }
+              }
+            }
+          }
+        },
+        "summary": "Get mail agent",
+        "description": "Get mail agent",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json:84]: @zohomail all operation descriptions should be different that the summary. Please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations).
Please fix all operations' descriptions in the swagger file.

File: certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json [hunk 3]
```diff
@@ -0,0 +1,800 @@
...
+        "operationId": "get_mail_agent",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json:85]: @zohomail all operationIds should be in Camel case, please change to "GetMailAgent". Please fix all operationIds in the swagger file.

File: certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json [hunk 4]
```diff
@@ -0,0 +1,800 @@
...
+    },
+    "/v1.0/email": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {}
```
Comment (Amjed-Ayoub) [inline certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json:148]: @zohomail empty schema is not allowed please remove or define a proper schema. please see [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/coding-standards#responses).
Please fix all empty schemas in the swagger file.

File: certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json [hunk 5]
```diff
@@ -0,0 +1,800 @@
...
+                  "x-ms-dynamic-list": {
+                    "operationId": "get_mail_agent",
+                    "itemValuePath": "mailagent_key",
+                    "itemsPath": "data",
+                    "itemTitlePath": "mailagent_name"
+                  }
+                },
+                "bounce_address": {
+                  "type": "string",
+                  "description": "Bounce addresss",
```
Comment (Amjed-Ayoub) [inline certified-connectors/Zoho ZeptoMail/apiDefinition.swagger.json:179]: @zohomail all files should be free of grammatical errors and typos. can you please change "addresss" to "address".

**Review by Amjed-Ayoub — CHANGES_REQUESTED** [review]
@zohomail Icon.png and setting.json files are not required in the open source repository. May I ask you please to remove them.
image
Thank you so much

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
@zohomail Thank you so much for fixing the PR. May I ask you please to sign our CLA, so we can approve and merge your PR?
You can sign it by committing the below command.

@microsoft-github-policy-service agree company="your company"
image


Please let me know if you have any questions.

Thank you so much again.

**Comment by zohomail** [comment]
@microsoft-github-policy-service agree company="Zoho Corporation"

**Review by Amjed-Ayoub — APPROVED** [review]

**Comment by Amjed-Ayoub** [comment]
Hello @zohomail,
I hope you are doing well.
Congratulations, your pull request is approved and merged. Please [submit the files to ISV Studio](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) or [update your submission](https://learn.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#update-submission-prior-to-certification) using your latest commit ID, please allow us up to 1-2 weeks to review your submission, and our Engineers will notify you in the “Activity Control” area in ISV Studio. Please [make sure that the icon meets all the requirements](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#design-an-icon-for-your-connector), that it’s size is 230x230, icon background is not white or transparent and it matches the iconBrandColor property in apiProperties.json file.
After making the submission in ISV Studio, please attach an intro.md file in the activity control section. This file is required and it's different from readme.md. You can [find an example of intro.md file on this page](https://docs.microsoft.com/en-us/connectors/custom-connectors/submit-for-certification#submit-to-isv-studio) at step 6.
Please [Create an environment in the Preview region](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#create-an-environment-in-the-preview-region). You will use this environment later to [test your connector](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-testing#test-your-connector) after Microsoft is done with the functional verification for your connector. 
We expect all tests to be completed within 1-2 weeks.
If your connector passes all tests, the deployment process begins, and it typically takes up to 3 to 4 weeks to deploy your connector to all of our regions.
Please let me know if you have any questions or run into any problems while creating or updating your submission.
Thank you very much for working with us.

**Pull request merged**